server/src/main/scala/actors/persistent/arrivals/AclForecastArrivalsActor.scala

```diff
@@ -32,7 +32,8 @@ class AclForecastArrivalsActor(initialSnapshotBytesThreshold: Int,
 
   override def handleFeedSuccess(incomingArrivals: Iterable[Arrival], createdAt: SDateLike): Unit = {
     log.info(s"Received arrivals (base)")
-    val incomingArrivalsWithKeys = incomingArrivals.map(a => (a.unique, a)).toMap
+    val incomingArrivalsWithKeys = incomingArrivals.map(a => (a.unique,
+      a.copy(TotalPax =  a.TotalPax ++ Set(TotalPaxSource(a.ActPax.getOrElse(0)-a.TranPax.getOrElse(0),AclFeedSource,None))))).toMap
```

Comment:
I'm not sure we should be removing transit passengers and using the result as the total passengers. We removed transit passengers from the total in other parts of the app. I think we should leave the total as the total